### PR TITLE
Budget-App-Project

### PR DESCRIPTION
**Pull request by omarsalem7**
## What does this PR do?

This PR is raised to review my budget app

## Description of the task to be completed
  - [x] Splash screen
  - [x] Sign up and log in pages
  - [x] Home page (catagories page)
  - [x] Transactions page
  - [x] "Add a new category" page
  - [x] "Add a new transaction" page
  
## How should it be tested

Clone the repository
```
git clone https://github.com/omarsalem7/Budget-app.git
```
Navigate to the folder
```
cd Budget-app
```
Checkout the branch
```
git checkout dev
```
> Video: [loom video](https://www.loom.com/share/42ebb27f4a144f11878491ec1c776296)
#### Thanks for reviewing my PR 😊

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,7 +52,6 @@ Feel free to check the [issues page](../../issues/).
 Give a ⭐️ if you like this project!
 
 ## Acknowledgments
+Original design idea by [Snapscan - iOs design and branding](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=)
```
Comment (micheaol) [inline README.md]: Please ensure that the README contains an "Acknowledgements" section that gives appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) according to the project requirement

File: README.md [hunk 2]
```diff
@@ -40,7 +52,6 @@ Feel free to check the [issues page](../../issues/).
 Give a ⭐️ if you like this project!
```
Comment (micheaol) [inline README.md:55]: Please ensure that you deploy the application using Heroku, Netlify, or GitHub Pages and, a link to the deployed project is present in the README according to the project requirement.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @omarsalem7 , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @omarsalem7, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 


## Required Changes ♻️
  - [ ] Your deployed app is not working. Kindly address the issue and fix it.
    Screen Shot 2022-06-02 at 12 34 21

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by omarsalem7** [comment]
Hi @rukundoeric,
Thanks for the feedback. I solved the issue, you can check now

**Review by arslanbisharat — APPROVED** [review]
Hi,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**